src/python/pants/backend/python/rules/coverage_integration_test.py

```diff
@@ -182,3 +183,29 @@ def test_coverage_with_filter(self) -> None:
             )
             in result.stderr_data
         )
+
+    def _assert_raw_coverage(self, result: PantsResult, build_root: str) -> None:
+        assert "Wrote raw coverage report to `dist/coverage/python`" in result.stderr_data
+        coverage_path = Path(build_root, "dist", "coverage", "python")
+        assert len(list(coverage_path.iterdir())) == 1
+        coverage_data = coverage_path / ".coverage"
+        assert coverage_data.exists() is True
+        conn = sqlite3.connect(coverage_data.as_posix())
+        cursor = conn.cursor()
+        cursor.execute("SELECT name FROM sqlite_master WHERE type='table';")
+        assert {row[0] for row in cursor.fetchall()} == {
+            "arc",
+            "context",
+            "coverage_schema",
+            "file",
+            "line_bits",
+            "meta",
+            "tracer",
+        }
+
+    def test_coverage_raw(self) -> None:
+        build_root = get_buildroot()
+        with temporary_dir(root_dir=build_root) as tmpdir:
+            tmpdir_relative = self._prepare_sources(tmpdir, build_root)
+            result = self._run_tests(tmpdir_relative, "--coverage-py-report=raw")
```

Comment:
Note that this is append syntax. Because there's a default, this is equivalent to `--coverage-py-report=['raw', 'console']`. That's fine, only want to make sure you're aware.

Reply:
If this is what you intend, it would be good to add `self._assert_console`. Fine to be in the followup that adds XML/HTML tests.